doc/dev/kclient_dev_bootstrap.rst

```diff
@@ -0,0 +1,376 @@
+        sudo ip link set tap0 master virbr0
+
+- ``ip link add name virbr0 type bridge``: Creates a new bridge interface
+  called ``virbr0``.
+- ``ip link set virbr0 up``: Brings the bridge interface online.
+- ``ip link set tap0 master virbr0``: Connects the ``tap0`` TAP interface to
+  the ``virbr0`` bridge, allowing network traffic to flow between the VM and
+  external networks.
+
+Launch the QEMU VM with TAP and vhost
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Start the VM with ``vhost=on`` to enable vhost for the TAP interface, improving
+packet processing performance.
+
+.. code-block:: bash 
+
+        sudo qemu-system-x86_64                                                                  \
+                                -enable-kvm                                                      \
+                                -m 4096                                                          \
+                                -cpu host                                                        \
+                                -netdev tap,id=net0,ifname=tap0,script=no,downscript=no,vhost=on \
+                                -device virtio-net-pci,netdev=net0                               \
+                                -drive file=$QCOW,format=qcow2                                   \
+                                -fsdev local,id=shared_folder,path=$SHR,security_model=none      \
+                                -device virtio-9p-pci,fsdev=shared_folder,mount_tag=hostshare    \
+                                -daemonize \
+                                -vnc :1
+
+- ``-netdev tap,id=net0,ifname=tap0,script=no,downscript=no,vhost=on``: Sets up
+  TAP networking with ``vhost`` enabled.
+- ``vhost=on``: offloads packet processing to the host kernel for improved
+  performance.
+- ``-device virtio-net-pci,netdev=net0``: Configures a ``virtio`` network device
+  for the VM, linked to the ``net0`` network backend.
+- ``-daemonize``: The qemu process runs in a background releasing the shell.
+
+Sepia Access and DNS Resolution
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The VMs on ``virbr0`` will have access to Sepia’s resources as if they were
+connected directly, benefiting from the `VPN`_ tunnel established on the host.
+The VMs will share the host’s DNS resolution, enabling seamless name resolution
+for addresses within the Sepia network.
+
+Build the Kernel
+----------------
+
+.. note:: The text assumes all prerequisites for building the kernel are
+   installed both on the host and guest.
+
+Clone the Git Repository
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Start by cloning the kernel source repository into the shared folder so that it
+is accessible from both the host and guest.
+
+.. code-block:: bash
+
+    git init linux && cd linux
+    git remote add torvalds git://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git
+    git remote add ceph https://github.com/ceph/ceph-client.git
+    git fetch && git checkout torvalds/master
```

Comment:
I had the issue here. The correct command:

git fetch --all &&  git checkout torvalds/master